docs/changelog_de.md

```diff
@@ -5,6 +5,7 @@
 * Fix #144: Die Rüstung "Gomez'Rüstung" heißt nun korrekt "Gomez' Rüstung".
 * Fix #149: Die Rüstung "verbesserte Erzrüstung" heißt nun korrekt "Verbesserte Erzrüstung".
 * Fix #192: Magier (NSCs, die nur mit Magie kämpfen) rüsten nicht länger automatisch Nah- und Fernkampfwaffen aus (z.B. nach dem Handeln). Dieser Fix setzt den Fix #59 voraus.
+* Fix #201: Die Beschreibung der Antiken Erzrüstung passt jetzt in die Textbox und enthält keinen Rechtschreibfehler mehr.
```

Comment:
See this [comment](https://github.com/AmProsius/gothic-1-community-patch/pull/200#discussion_r598336425).